catalog/core/catalog-core-ftp/src/main/java/ddf/catalog/core/ftp/FtpServerStarter.java

```diff
@@ -146,6 +222,21 @@ protected Listener getDefaultListener() {
         return ((DefaultFtpServer) server).getListener(DEFAULT_LISTENER);
     }
 
+    public void setPort(int port) {
+        this.port = port;
+    }
+
+    public void setImplicitMode(boolean useImplicitSecurity) {
+        LOGGER.warn(
```

Comment:
I don't think this log statement has much value here - but this information should be in the description of the metatype.